vowpalwabbit/ccb_label.h

```diff
@@ -5,17 +5,18 @@
 #pragma once
 
 #include <cstdint>
+#include <string>
 #include <vector>
 
 #include "action_score.h"
+#include "ccb_reduction_features.h"
 #include "label_parser.h"
 #include "v_array.h"
-// TODO: This header can be removed once type and explicit_included_actions are removed from the label
-#include "ccb_reduction_features.h"
-#include "fmt/format.h"
-#include "io_buf.h"
 #include "vw_string_view.h"
 
+// TODO: This header can be removed once type and explicit_included_actions are removed from the label
```

Comment:
Does this comment go with `ccb_reduction_features.h`?

Reply:
Looks like it. Seems that clang ignored the association. Ill update the comment